activerecord/lib/active_record/attribute_methods.rb

```diff
@@ -9,7 +9,7 @@ module AttributeMethods
     include ActiveModel::AttributeMethods
 
     included do
-      initialize_generated_modules
+      _attribute_methods_builder
```

Comment:
@rafaelfranca I've removed the memoization resetter here, got around the problem by explicitly including `AttributeMissingMethods` in AM::AttributeMethods.

Actually specs pass without this line as well, but I'm leaving in because I think it's safer all-around to ensure that the module is immediately included in the ancestors, rather than waiting for the first call to prefix/suffix/define_attribute_methods.